core/HomeButton.java

```diff
@@ -0,0 +1,46 @@
+package edu.asu.c3simulator.widgets;
```

Comment:
The package is declared properly, but the file is in the wrong folder. Make sure it is in its package directory.